src/main/resources/hudson/plugins/active_directory/ActiveDirectoryStatus/index.jelly

```diff
@@ -2,7 +2,7 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
     <l:layout title="${it.displayName}">
         <l:main-panel>
-            <h1><img style="width:32px; height:32px" src="${resURL}/plugin/active-directory/images/icon.png"/> ${it.displayName}</h1>
+            <h1><l:icon src="symbol-medkit-outline plugin-ionicons-api" class="icon-xlg" /> ${it.displayName}</h1>
```

Comment:
Should sub-pages have an icon in the `<h1>` tag at all? Not sure what current guidance is from @jenkinsci/sig-ux.